tests/ReplayData/BranchProtection2.setUp.txt

```diff
@@ -0,0 +1,33 @@
+https
```

Comment:
I think this file is not needed any more, right?

Reply:
Done in c80d8eab2748b30e85e3ba19873c7033c43403c3